src/components/Match/Overview/Timeline.jsx

```diff
@@ -0,0 +1,302 @@
+            <div className={styles.line}>
+              <section style={{ width: '100%' }} />
+            </div>
+            <div className={styles.events}>
+              <mark
+                className={styles.battleHorn}
+                style={{
+                  left: `${(preHorn * 100) / (match.duration + preHorn)}%`,
+                }}
+              />
+              {
+                obj[0].map((obj, i) => {
+                  const side = (
+                    obj.player_slot && isRadiant(obj.player_slot))
+                      || (obj.team && obj.team === 2
+                  ) ? 'radiant' : 'dire';
+                  const wTeamfight = obj.type === 'teamfight' && (100 * (obj.end - obj.start)) / (match.duration + preHorn);
+
+                  return (
+                    <mark
+                      key={i}
+                      className={obj.type === 'teamfight' ? styles.teamfight : styles[side]}
+                      style={{
+                        left: obj.time && `${
+                          ((100 * (obj.time + preHorn)) / (match.duration + preHorn)) - (wTeamfight / 2)
+                        }%`,
+                        width: `${wTeamfight}%`,
+                        // backgroundColor: obj.type === 'teamfight' && (
+                        //   obj.radiant_gold_delta >= 0 ? styles.green : styles.red
+                        // ),
+                      }}
+                    >
+                      {obj.type === 'firstblood' &&
+                        <IconBloodDrop
+                          data-tip
+                          data-for={`event_${i}`}
+                        />
+                      }
+                      {obj.type === 'roshan' &&
+                        <IconRoshan
+                          data-tip
+                          data-for={`event_${i}`}
+                        />
+                      }
+                      {obj.type === 'teamfight' &&
+                        <IconBattle
+                          data-tip
+                          data-for={`event_${i}`}
+                          style={{ fill: obj.radiant_gold_delta >= 0 ? styles.green : styles.red }}
+                        />
+                      }
+                      <ReactTooltip
+                        id={`event_${i}`}
+                        effect="solid"
+                        place="right"
+                      >
+                        {obj.type === 'firstblood' &&
+                          <section>
+                            {match.players
+                              .filter(player => player.player_slot === obj.player_slot)
+                              .map(player => (
+                                <aside style={{ color: playerColors[obj.player_slot] }}>
+                                  <img
+                                    src={`${API_HOST}/apps/dota2/images/heroes/${heroes[player.hero_id].name.split('npc_dota_hero_')[1]}_icon.png`}
```

Comment:
Did you also want to swap to this in the minimap tooltips (don't remember if you ended up using the minimap icons?)